From 106a6f66e9b552b15889585e4cbc25bf51ef2c47 Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@nxp.com>
Date: Tue, 23 Aug 2016 17:13:35 -0500
Subject: [PATCH 034/107] staging: fsl-mc: move dprc-driver private definitions
 out of mc-bus.h
Git-commit: 2b0011d2bf5924f1509903f15bcb3359521c42bb
Patch-mainline: v4.9-rc1
References: bsc#1041022

Move definitions that are private to dprc-driver.c out of the
public mc-bus.h header.

Signed-off-by: Stuart Yoder <stuart.yoder@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/staging/fsl-mc/bus/dprc-driver.c | 6 ++++++
 drivers/staging/fsl-mc/include/mc-bus.h  | 6 ------
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/dprc-driver.c b/drivers/staging/fsl-mc/bus/dprc-driver.c
index 576244b..5e34720 100644
--- a/drivers/staging/fsl-mc/bus/dprc-driver.c
+++ b/drivers/staging/fsl-mc/bus/dprc-driver.c
@@ -18,6 +18,12 @@
 #include "dprc-cmd.h"
 #include "fsl-mc-private.h"
 
+#define FSL_MC_DPRC_DRIVER_NAME    "fsl_mc_dprc"
+
+#define FSL_MC_DEVICE_MATCH(_mc_dev, _obj_desc) \
+	(strcmp((_mc_dev)->obj_desc.type, (_obj_desc)->type) == 0 && \
+	 (_mc_dev)->obj_desc.id == (_obj_desc)->id)
+
 struct dprc_child_objs {
 	int child_count;
 	struct dprc_obj_desc *child_array;
diff --git a/drivers/staging/fsl-mc/include/mc-bus.h b/drivers/staging/fsl-mc/include/mc-bus.h
index 8a32e55..3f80506 100644
--- a/drivers/staging/fsl-mc/include/mc-bus.h
+++ b/drivers/staging/fsl-mc/include/mc-bus.h
@@ -15,12 +15,6 @@
 #include <linux/mutex.h>
 #include <linux/stringify.h>
 
-#define FSL_MC_DPRC_DRIVER_NAME    "fsl_mc_dprc"
-
-#define FSL_MC_DEVICE_MATCH(_mc_dev, _obj_desc) \
-	(strcmp((_mc_dev)->obj_desc.type, (_obj_desc)->type) == 0 && \
-	 (_mc_dev)->obj_desc.id == (_obj_desc)->id)
-
 struct irq_domain;
 struct msi_domain_info;
 
-- 
1.8.5.6

